packages/switch/test/lion-button-switch.test.js

```diff
@@ -0,0 +1,146 @@
+import { expect, fixture, html } from '@open-wc/testing';
+import sinon from 'sinon';
+
+import { LionButton } from '@lion/button';
+
+import '../lion-button-switch.js';
+
+describe('lion-button-switch', () => {
+  let el;
+  beforeEach(async () => {
+    el = await fixture(html`
+      <lion-button-switch></lion-button-switch>
+    `);
+  });
+
+  it('should behave as a button', () => {
+    expect(el instanceof LionButton).to.be.true;
+  });
+
+  it('should have tabindex="0"', () => {
```

Comment:
maybe `should be focusable` to make it read more like english and to communicate better to non-devs (which is one of the strives of BDDs)